Tamil/Tamil_Thai_Vazhtu.md

```diff
@@ -0,0 +1,27 @@
+நீராரும் கடல் உடுத்த நில மடந்தைக் கெழிலொழுகும்
```

Comment:
It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.